test/service/test_manifest.py

```diff
@@ -272,6 +272,24 @@ class DCP1ManifestTestCase(ManifestTestCase, DCP1CannedBundleTestCase):
     pass
 
 
+class ReplicaBasedManifestTestCase(ManifestTestCase, metaclass=ABCMeta):
```

Comment:
I think we can provide this fixture at a higher level in the class hierarchy, in an existing `…TestCase` class without complicating the class hierarchy with this mixin.